### PR TITLE
Add New Readme

### PR DESCRIPTION
**Pull request by mlieberman85**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,134 +1,65 @@
...
+See LICENSE.MD
 
-```bash
-./target/release/real_project
-```
 
-You should get a "Goodbye, World" output on the hijacked ones.
 
-### Build in Container
+<p align="right">(<a href="#top">back to top</a>)</p>
```
Comment (rgreinho) [inline README.md:64]: A bit nitpicking, but a markdown anchor would do the trick rather than embedding HTML into the markdown.

**Review by rgreinho — COMMENTED** [review]
No blocker from me, just a small comment 👍

File: README.md (hunk 1 above)
Comment (mlieberman85) [reply README.md:64]: Oops. Yeah, I took it from a template. I'm going to fix that .

**Review by rgreinho — APPROVED** [review]
Approved 👍

**Review by bradbeck — APPROVED** [review]

**Pull request merged**